x-pack/plugins/siem/common/types/timeline/index.ts

```diff
@@ -136,6 +133,17 @@ const SavedSortRuntimeType = runtimeTypes.partial({
 /*
  *  Timeline Types
  */
+
+export enum TimelineType {
```

Comment:
nit -> maybe interesting for this case https://github.com/gcanti/io-ts/issues/67#issuecomment-368068197